src/name_bazaar/ui/core.cljs

```diff
@@ -47,9 +47,14 @@
                    {:async-flow {:first-dispatch [:district0x/load-smart-contracts {:version constants/contracts-version}]
                                  :rules [{:when :seen?
                                           :events [:district0x/smart-contracts-loaded :district0x/my-addresses-loaded]
-                                          :dispatch-n [[:district0x/watch-my-eth-balances]
-                                                       [:active-page-changed]
-                                                       [:try-resolving-address]]}]}
+                                          :dispatch-n (let [events [[:district0x/watch-my-eth-balances]
```

Comment:
This can be written in nicer way like this I think: 
```clojure
(remove nil?
                                            [[:district0x/watch-my-eth-balances]
                                             [:try-resolving-address]
                                             (when history/hashroutes?
                                               [:active-page-changed])])
```

Also, I think `:try-resolving-address` might not be needed as well. you should try url `/user/namebazaar/offerings` if you see offerings with server side render